shark/iree_utils.py

```diff
@@ -89,12 +89,23 @@ def get_iree_gpu_args():
     else:
         return ["--iree-hal-cuda-disable-loop-nounroll-wa"]
 
-def get_iree_vulkan_args():
-    return [
-        "--iree-flow-demote-i64-to-i32=false",
-        "--iree-flow-demote-f64-to-f32=true"
-    ]
+def get_vulkan_triple_flag():
+    vulkan_device_cmd = "vulkaninfo | grep deviceName | awk \'END{{print $3}}\'"
+    vulkan_device = run_cmd(vulkan_device_cmd).strip()
+    if vulkan_device == "apple":
+        return "-iree-vulkan-target-triple=m1-moltenvk-macos"
+    elif vulkan_device == "A100-SXM4-40GB":
+        return "-iree-vulkan-target-triple=ampere-rtx3080-linux"
+    else:
+        print("Optimized kernel for your target device is not added yet. Contact SHARK Admin on discord or pull up an issue.")
```

Comment:
is there a slow default we can fall into ? and also maybe insert the discord link for easy access.

Reply:
It is already does go into slow default by not adding a vulkan-target-triple flag. But yeah, I can add the link into it. :)